lib/api/apiUtils/object/partInfo.js

```diff
@@ -20,13 +20,23 @@ function getPartNumber(query) {
  */
 function getPartSize(objMD, partNumber) {
     let size;
+    let locationPartNumber;
     if (partNumber && objMD && objMD.location
         && objMD.location.length >= partNumber) {
         const locations = [];
         for (let i = 0; i < objMD.location.length; i++) {
             const { dataStoreETag } = objMD.location[i];
-            const locationPartNumber =
+            if (!dataStoreETag) {
```

Comment:
May be reverse the condition, `if (dataStoreETag) {} else {}` would look cleaner.

Reply:
Agreed